folding-schemes/src/frontend/noname/mod.rs

```diff
@@ -0,0 +1,228 @@
+use crate::Error;
+use ark_noname::sonobe::NoNameSonobeCircuit;
+use ark_r1cs_std::alloc::AllocVar;
+use ark_r1cs_std::fields::fp::FpVar;
+use ark_relations::r1cs::{ConstraintSynthesizer, SynthesisError};
+use num_bigint::BigUint;
+use std::marker::PhantomData;
+
+use super::FCircuit;
+use ark_ff::PrimeField;
+use ark_noname::utils::compile_source_code;
+use noname::backends::{r1cs::R1CS as R1CSNoName, BackendField};
+use noname::witness::CompiledCircuit;
+
+pub mod utils;
+
+use utils::*;
+
+#[derive(Debug, Clone)]
+pub struct NoNameFCircuit<F: PrimeField, BF: BackendField> {
```

Comment:
is it `NoName` or `Noname`? from the repo readme it seems to be the latter (https://github.com/zksecurity/noname)

Reply:
Changed all occurrences as requested